icons/drum.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "drummer",
```

Comment:
Also, kind looks like an audio jack.

Reply:
> Also, kind looks like an audio jack.

?

Reply:
Maybe a `socket` tag could be added, idk.